pyramid/metrics.go

```diff
@@ -0,0 +1,41 @@
+package pyramid
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// nolint: gomnd
+const (
+	kb          = float64(1024)
+	mb          = 1024 * kb
+	fsNameLabel = "fsName"
+)
+
+var cacheHit = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "tier_fs_cache_hits_total",
+		Help: "TierFS cache hits total count",
+	}, []string{fsNameLabel})
+
+var cacheMiss = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "tier_fs_cache_miss_total",
+		Help: "TierFS cache miss total count",
+	}, []string{fsNameLabel})
+
+var evictionHistograms = promauto.NewHistogramVec(
+	prometheus.HistogramOpts{
+		Name:    "tier_fs_eviction_bytes",
+		Help:    "TierFS evicted object size by bytes",
+		Buckets: []float64{0.5 * kb, 1 * kb, 16 * kb, 32 * kb, 128 * kb, 512 * kb, 1 * mb, 2 * mb, 4 * mb, 8 * mb, 16 * mb, 64 * mb},
```

Comment:
It will be hard to compare buckets of these sizes, they are not equally-spaced on a log scale.  The multipliers between successive elements should be constant, here they are 2, 16, 2, 4, 4, 2, 2, 2, 2, 2, 4.  I think [`ExponentialBuckets`](https://godoc.org/github.com/prometheus/client_golang/prometheus#ExponentialBuckets) does exactly that.